homeassistant/util/temperature.py

```diff
@@ -27,3 +29,28 @@ def convert(temperature: float, from_unit: str, to_unit: str) -> float:
     elif from_unit == TEMP_CELSIUS:
         return celsius_to_fahrenheit(temperature)
     return fahrenheit_to_celsius(temperature)
+
+
+def calculate_dewpoint(temperature: float, humidity: float,
```

Comment:
This is not common enough that we need to put this in the main util class. Let's keep it in the sensor.